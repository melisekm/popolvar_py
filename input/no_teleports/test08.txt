7 7 47
CCCHHCH
CCHHPCH
HCDCHCC
HHCCHHP
HCHHCCC
HCHHPHH
PCHCHHP