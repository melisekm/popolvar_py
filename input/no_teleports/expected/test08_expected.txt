0 0
1 0
1 1
1 2
2 2
3 2
3 1
4 1
5 1
5 2
6 2
6 3
6 4
6 5
6 6
5 6
5 5
4 5
3 5
3 6
2 6
1 6
0 6